Library/Formula/go.rb

```diff
@@ -76,9 +93,10 @@ def caveats; <<-EOS.undent
     `go vet` and `go doc` are now part of the go.tools sub repo:
       http://golang.org/doc/go1.2#go_tools_godoc
 
-    To get `go vet` and `go doc` run:
+    `godoc` has been installed for you unless you requested otherwise.
+
+    To get `go vet` run:
       go get golang.org/x/tools/cmd/vet
-      go get golang.org/x/tools/cmd/godoc
```

Comment:
I think it's generally worth doing; I hate computers being ambiguous when they know what I've done. In this case I think we can just avoid mention of either if we're installing by default.